.azure-pipelines/Windows-CI.yml

```diff
@@ -62,6 +56,7 @@ jobs:
       git submodule update --init --recursive
       set ONNX_BUILD_TESTS=1
       set ONNX_ML=$(onnx_ml)
+      set ONNX_VERIFY_PROTO_3=$(onnx_verify_proto)
```

Comment:
It might just be accidentally removed somehow... Adding it back to cover more cases.